components/molecule/notification/src/_settings.scss

```diff
@@ -2,10 +2,17 @@ $w-notification: 100% !default;
 $z-notification: 1000 !default;
 $mh-notification: 500px !default;
 $sz-notification-icon: $sz-icon-m !default;
-$ai-notification-content: normal !default;
+$ai-notification-content: center !default;
```

Comment:
Isn't this a breaking change? Couldn't you leave it at it was and change it in your theme?

Reply:
ok